Header header
int32 position
bool has_error
bool is_homed
bool is_active

uint8 command_type
int32 command_value
int32 max_speed
int32 acceleration
